.github/workflows/tests.yaml

```
@@ -0,0 +1,56 @@
+name: tests
+
+on: [pull_request, push]
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - uses: ljharb/actions/node/install@main
+        name: 'nvm install ${{ matrix.node-version }} && npm install'
+        with:
+          node-version: ${{ matrix.node-version }}
+          skip-ls-check: true
```

this one doesn't actually need a matrix :-)
```suggestion
    steps:
      - name: Checkout
        uses: actions/checkout@v3

      - uses: ljharb/actions/node/install@main
        name: 'nvm install lts/* && npm install'
        with:
          node-version: lts/*
          skip-ls-check: true
```

---

Decided to add for consistency 😄